jerry-core/CMakeLists.txt

```diff
@@ -62,7 +62,7 @@ message(STATUS "MEM_HEAP_SIZE_KB          " ${MEM_HEAP_SIZE_KB})
 set(INCLUDE_CORE
     "${CMAKE_CURRENT_SOURCE_DIR}"
     "${CMAKE_CURRENT_SOURCE_DIR}/api"
-    "${CMAKE_CURRENT_SOURCE_DIR}/debugger"
+    "${CMAKE_CURRENT_SOURCE_DIR}/dbg"
```

Comment:
do we agree at least that jerry_ prefix should not be used for non-public symbols?